config/inline_internal_test.go

```diff
@@ -0,0 +1,265 @@
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclparse"
+)
+
+func TestInline_Backend(t *testing.T) {
```

Comment:
I think those inline tests have not the value they could have. They just tests that `go` works with returning the inline type definition. I still can add `asdf = 1` attribute to a backend definition without a schema error.